res/linux/15-mixxx-usb.rules

```diff
@@ -0,0 +1,11 @@
+# This udev rule allows Mixxx to access HID and USB Bulk controllers when running as a normal user
```

Comment:
To work with dh_installudev which I believe is what we're supposed to be using, it needs to be named ```mixxx.usb.rules``` not ```mixxx-usb.rules```.

http://manpages.ubuntu.com/manpages/intrepid/man1/dh_installudev.1.html